lib/ansible/galaxy/collection.py

```diff
@@ -356,6 +378,45 @@ def from_name(collection, apis, requirement, force, parent=None):
         return req
 
 
+def get_available_api_versions(galaxy_api):
+    headers = {}
+    headers.update(galaxy_api._auth_header(required=False))
```

Comment:
you could just do `headers = galaxy_api._auth_header(required=False)`, not much need for .update here